src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/commands/mobile/set_interior_vehicle_data_request.cc

```diff
@@ -148,168 +170,184 @@ const std::string LightName(const smart_objects::SmartObject& light_name) {
   return ok ? name : "unknown";
 }
 
-bool CheckLightDataByCapabilities(
+mobile_apis::Result::eType GetItemCapability(
     const smart_objects::SmartObject& capabilities,
```

Comment:
Is it the whole RC CAP, or the light CAP

Reply:
@yang1070 it is caps corresponding to the module type,
in case when we use this function for light module, than it will be light caps